Node/src/models/o365ConnectorCard.ts

```diff
@@ -166,6 +166,18 @@ export class O365ConnectorCardSection implements teams.IIsO365ConnectorCardSecti
         return this;
     }
 
+    public activityImageType(imageType: string): this {
+        if (imageType) {
+            if (imageType === 'avatar' || imageType === 'article')
```

Comment:
'avatar' and 'article' should be defined as constants, perhaps like ActivityTypes in C#. Discoverability of the valid values for these strings is already a problem.